### PR TITLE
[Fix]: Add header feature in latest v14.

### PR DESCRIPTION
**Pull request by hkapatel-initos**
Add header feature in latest v14

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
Lgtm. Where is it used ?

**Comment by hkapatel-initos** [comment]
> Lgtm. Where is it used ?

This feature is used in sql export_excel #2115 module the header parameter was removed but it is added back in this PR to avoid losing the feature.

**Review by florian-dacosta — APPROVED** [review]
@legalsylvain 
It was initially added here in v12 : https://github.com/OCA/server-tools/pull/1659
But, the before it was merged, the v13 PR of sql_request_abstract was created and this small addition was lost.

The goal is for sql_export_excel to be able to add or not the header in the generated xlsx file.
(It can be usefull not not to add it because the sql_export_excel file allow to inject the result of the query inside an existing xlsx file/template)

**Comment by legalsylvain** [comment]
Hi @florian-dacosta Thanks for the explanation.

/ocabot merge patch

**Comment by legalsylvain** [comment]
> Sorry @legalsylvain you are not allowed to merge.

that's not very kind of you, dear bot.

Could you approve https://github.com/OCA/server-tools/pull/2146 ?

**Review by dsolanki-initos — APPROVED** [review]
LGTM

File: sql_request_abstract/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "SQL Request Abstract",
-    "version": "14.0.1.0.0",
+    "version": "14.0.1.0.1",
```
Comment (legalsylvain) [inline sql_request_abstract/__manifest__.py:7]: for your information, that change is not necessary, as  the  oca bot bump the version.

**Pull request merged**